rapid/src/main/java/com/vrg/rapid/ConsistentHash.java

```diff
@@ -0,0 +1,64 @@
+package com.vrg.rapid;
+
+import com.vrg.rapid.pb.Endpoint;
+import net.openhft.hashing.LongHashFunction;
+
+import javax.annotation.Nullable;
+import java.util.Collection;
+import java.util.SortedMap;
+import java.util.TreeMap;
+
+/**
+ * Simple consistent Hash implementation, adapted for Endpoints
+ *
+ * https://tom-e-white.com/2007/11/consistent-hashing.html
+ */
+public class ConsistentHash<T> {
```

Comment:
What is the rationale for adding another consistent hash based ring implementation? Wouldn't any of the K rings in the MembershipView data structure suffice?